lib/create-ui-config-api.ts

```diff
@@ -0,0 +1,62 @@
+import { MakeRequest } from './common-types'
+import entities from './entities'
+import { UIConfig } from './entities/ui-config'
+
+/**
+ * @private
+ */
+export type ContentfulUIConfigApi = ReturnType<typeof createUIConfigApi>
+
+/**
+ * @private
+ */
+export default function createUIConfigApi(makeRequest: MakeRequest) {
+  const { wrapUIConfig } = entities.uiConfig
+
+  const getParams = (self: UIConfig) => {
+    const uiConfig = self.toPlainObject()
+
+    return {
+      params: {
+        spaceId: uiConfig.sys.space.sys.id,
+        environmentId: uiConfig.sys.environment.sys.id,
+      },
+      raw: uiConfig,
+    }
+  }
+
+  return {
+    /**
+     * Sends an update to the server with any changes made to the object's properties
+     * @return Object returned from the server with updated changes.
+     * @example ```javascript
+     * const contentful = require('contentful-management')
+     *
+     * const client = contentful.createClient({
+     *   accessToken: '<content_management_api_key>'
+     * })
+     *
+     * client.getSpace('<space_id>')
+     * .then((space) => space.getEnvironment('<environment_id>'))
+     * .then((environment) => environment.getUIConfig('<entry_id>'))
```

Comment:
I don't think we need an entry-id here right?

Reply:
Copy paste :D